main.py

```diff
@@ -129,15 +133,11 @@ def register():
 	try:
 		db.session.add(app_data)
 		db.session.commit()
-		statsd.increment('application_registered.success',
-		                 tags=['app_id:%s' % app_id])
-		logger.info('application_registered.success: (%d app_id)', app_id)
+		short_log('application_registered.success', app_id)
 		return app_id
 	except IntegrityError:
 		db.session.rollback()
-		statsd.increment('application_registered.failed',
-		                 tags=['email:%s' % email])
-		logger.error('application_registered.failed: (%d app_id)', app_id)
```

Comment:
you used "logger.error" in the past, but now you use "logger.info" this will make it harder to search the logs for errors